catalogs/data/airbyte/terraform/azure/postgres.tf

```diff
@@ -0,0 +1,60 @@
+data "plural_service_context" "dev_network" {
+  name = "plrl/network/dev"
```

Comment:
this should be parameterized (and might be part of the issue here).  It's always being put in the dev network right now.

Reply:
I still have to do it, but dev and prod should be the same now. Which tier mgmt cluster should be in? Should it be manual step?

Reply:
It's done now. It reads `tier` tag from the cluster and if there is none it will be defaulted to `dev`.